Mathlib/Order/KrullDimension.lean

```diff
@@ -290,6 +406,10 @@ lemma coe_lt_height_iff {x : α} {n : ℕ} (hfin : height x < ⊤) :
   mpr := fun ⟨y, hyx, hy⟩ =>
     hy ▸ height_strictMono hyx (lt_of_le_of_lt (height_mono hyx.le) hfin)
 
+lemma coe_lt_coheight_iff {x : α} {n : ℕ} (hfin : coheight x < ⊤):
+    n < coheight x ↔ (∃ y > x, coheight y = n) :=
```

Comment:
```suggestion
    n < coheight x ↔ ∃ y > x, coheight y = n :=
```

Reply:
(Personally, I’d be inclined to ban omitting parenthesis on the right of a symmetric operator when you cannot omit them on the left.)